lib/Github/Api/PullRequest.php

```diff
@@ -58,6 +58,13 @@ public function files($username, $repository, $id)
         return $this->get('repos/'.rawurlencode($username).'/'.rawurlencode($repository).'/pulls/'.rawurlencode($id).'/files');
     }
 
+    public function status($username, $repository, $id)
+    {
+        $link = $this->show($username, $repository, $id)['_links']['statuses']['href'];
```

Comment:
Yeah it just to help get the status response for a PR since its URL is built from a SHA instead of the PR Number like all of the other URLs